pom.xml

```diff
@@ -97,14 +97,14 @@
         <dependency>
             <groupId>org.xhtmlrenderer</groupId>
             <artifactId>flying-saucer-pdf</artifactId>
-            <version>9.8.0</version>
+            <version>9.11.1</version>
```

Comment:
THere seems to be a [9.11.2](https://github.com/flyingsaucerproject/flyingsaucer/releases/tag/v9.11.2)